helpers/deploymentConfig.ts

```diff
@@ -2509,27 +2509,15 @@ export const globalConfig: NetworkConfig = {
         rewards: [
           {
             asset: "XVS",
-            markets: ["WBTC"],
-            supplySpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
-            borrowSpeeds: ["190258751902587"], // 190258751902587 XVS over 365 days (2628000 blocks)
+            markets: ["WETH","WBTC", "USDT", "USDC","crvUSD"],
+            supplySpeeds: [ "27777777777777","83333333333333","83333333333333","83333333333333","37037037037036"],
+            borrowSpeeds: ["41666666666666","125000000000000","125000000000000","125000000000000","55555555555555"],
```

Comment:
Can we mention the number of tokens, duration and total blocks for rewards in comments as we are doing for other networks?